WordPress/WordPressTest/ContextManagerTests.swift

```diff
@@ -5,74 +5,64 @@ import CoreData
 @testable import WordPress
 
 class ContextManagerTests: XCTestCase {
```

Comment:
The diff in this test case isn't very straight forward to see the actual changes. I thought I'll explain a bit more about it. The structure was changed, like how an old database is created and how the migration is performed. But the test code aren't, like inserting data into database and making assertions that the data is still there after migration.